tools/DevDiagnostics/DevHome.DevDiagnostics/Helpers/ImageOptionsInternalTool.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using DevHome.DevDiagnostics.Models;
+using DevHome.DevDiagnostics.Properties;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+
+namespace DevHome.DevDiagnostics.Helpers;
+
+public class ImageOptionsInternalTool : Tool
+{
+    private const string ButtonText = "\ue9d5";
+    private static readonly string _toolName = CommonHelper.GetLocalizedString("ImageOptionsName");
+
+    public ImageOptionsInternalTool()
+        : base(_toolName, ToolType.Unknown, Settings.Default.IsImageOptionsToolPinned)
+    {
+    }
+
+    public override IconElement GetIcon()
+    {
+        return new FontIcon
+        {
+            Glyph = ButtonText,
+            FontFamily = (FontFamily)Application.Current.Resources["SymbolThemeFontFamily"],
+        };
+    }
+
+    internal override void InvokeTool(ToolLaunchOptions options)
+    {
+        if (TargetAppData.Instance.TargetProcess == null || TargetAppData.Instance.TargetProcess.MainModule == null)
+        {
+            return;
+        }
+
+        try
+        {
+            FileInfo fileInfo = new FileInfo(Environment.ProcessPath ?? string.Empty);
+            Directory.SetCurrentDirectory(fileInfo.DirectoryName ?? string.Empty);
+
+            var startInfo = new ProcessStartInfo()
+            {
+                FileName = "DevHome.IfeoTool.exe",
```

Comment:
This should be the full path to the WindowsApps\DevHome_PFN\devhome.ifeotool.exe, otherwise it can end up launching the wrong version if you have two versions of devhome installed(Canary and Preview) or not launch at all if the app execution alias is disabled for some reason.

Reply:
fixed in latest iteration.